<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>424</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MaxFlow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number Of Edges :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Number_Nodes">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>110</width>
      <height>13</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;From&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linet_from">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linet_T">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineT_cost">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddNode">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Max_flow">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Flow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Get_paths">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get All paths</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Answer">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>210</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Hint : Base of the graph is zero &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Enter graph</string>
    </property>
    <addaction name="actionAdd_Number_of_Nodes"/>
   </widget>
   <widget class="QMenu" name="menuOnline_Graph">
    <property name="title">
     <string>Online </string>
    </property>
    <addaction name="actionOnline_Graph"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionAbout_Developer_2"/>
    <addaction name="actionAbout_Program"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuOnline_Graph"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Number_of_Nodes">
   <property name="text">
    <string>Add Number of Nodes</string>
   </property>
  </action>
  <action name="actionOnline_Graph">
   <property name="text">
    <string>Online Graph</string>
   </property>
  </action>
  <action name="actionAbout_Developer">
   <property name="text">
    <string>about Developer</string>
   </property>
  </action>
  <action name="actionAbout_Developer_2">
   <property name="text">
    <string>about Developer</string>
   </property>
  </action>
  <action name="actionAbout_Program">
   <property name="text">
    <string>about Program</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
